### PR TITLE
Improved Readme in CLI and EnvRegistry

### PR DESCRIPTION
**Pull request by dmitriyslaym**

### CODE REVIEW

File: packages/service-environment-registry/README.md [hunk 1]
```diff
@@ -37,7 +37,10 @@ You can get the default export from the link
           "environments$": { "asyncModel": "requestStream" }
         }
       },
-      "config": { "serviceName": "EnvironmentRegistryService" }
+      "config": { 
+        "serviceName": "EnvironmentRegistryService",
+        "token": "workspace"
```
Comment (stephanebenayoun) [inline packages/service-environment-registry/README.md]: If I'm not wrong, the value of `token` here could be anything right ? In this case putting 'workspace' is a little bit misleading

File: packages/cli/README.md [hunk 2]
```diff
@@ -67,6 +76,25 @@ capsulahub build --help
...
+
+### HttpFile
+
+Make sure, that you have httpServer running on the port 1111 and you have
+a configuration file in `./configuration/workspace.json` (the path is relative to the root of your server).
+
+In order to run the application locally run:
+
+```bash
+capsulahub run --token=http://localhost:1111/configuration --port=8888
```
Comment (MikeSquall) [inline packages/cli/README.md]:
```suggestion
capsulahub run --token=http://localhost:1111/configuration
```

Comment (MikeSquall) [reply packages/cli/README.md]: No need to specify port here IMO

File: packages/cli/README.md [hunk 3]
```diff
@@ -67,6 +76,25 @@ capsulahub build --help
...
+```
+
+If you want to generate a build with html and js file in it, you should run:
```
Comment (MikeSquall) [inline packages/cli/README.md]:
```suggestion
If you want to generate a bundle containing the app in order to serve it by yourself, you should run:
```

File: packages/cli/README.md (hunk 2 above)
Comment (dmitriyslaym) [reply packages/cli/README.md]: It's not required, so we can omit it. I just like 8888 more)

File: packages/service-environment-registry/README.md (hunk 1 above)
Comment (dmitriyslaym) [reply packages/service-environment-registry/README.md]: what do you suggest to put here? it's an example of usage, so I copied the real config

**Pull request merged**